lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlBasic.pm

```diff
@@ -0,0 +1,97 @@
+package Genome::Model::Tools::Analysis::DumpIgvXmlBasic;
+
+use strict;
+use warnings;
+
+use Genome;
+use Command;
+use IO::File;
+use Cwd qw(abs_path);
+use URI;
+use XML::Writer;
+
+class Genome::Model::Tools::Analysis::DumpIgvXmlBasic {
+    is => 'Command',
+    has => [
+        resource_files => {
+            type => 'HASH',
```

Comment:
This has been addressed by setting _is_hidden_in_docs to 1 which results in this command not showing up in the docs/autocomplete of the command line. People can still try to execute it on the command line but they would need to know that it exists.